pages/common/gh-workflow.md

```diff
@@ -17,19 +17,19 @@
 
 - Display the YAML definition for a specific Git branch or tag:
 
-`gh workflow view {{id|workflow_name|filename.yml}} --ref {{branch_or_tag_name}} --yaml`
+`gh workflow view {{id|workflow_name|filename.yml}} --ref {{branch|tag_name}} --yaml`
 
 - List workflow files (use `--all` to include disabled workflows):
 
 `gh workflow list`
 
 - Run a manual workflow with parameters:
 
-`gh workflow run {{id|workflow_name|filename.yml}} --raw-field {{param1}}={{value1}} --raw-field {{param2}}={{value2}}`
+`gh workflow run {{id|workflow_name|filename.yml}} {{--raw-field param1=value1 --raw-field param2=value2 ...}}`
```

Comment:
`--raw-field` is not a placeholder. I prefer the original syntax.

Reply:
Any updates on this @EmilySeville7cfg?

Reply:
As it's not specified whether placeholder apply to options or just to their arguments as long as positional ones do what u think suites the best for the page. It's not a problem to leave this line as it was (as this change is not required).